app/scss/questionPage.scss

```diff
@@ -35,6 +35,10 @@ body {
   }
 }
 
+.hide-toggle-fullscreen {
```

Comment:
there is already a bootstrap class that will do this for you:
https://getbootstrap.com/docs/4.6/utilities/display/#hiding-elements